model/versioning/VersionedGridFieldDetailForm.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+/**
+ * Extends {@see GridFieldDetailForm}
+ */
+class VersionedGridFieldDetailForm extends Extension {
```

Comment:
Because if it's core, it cannot be substituted or removed.

Versioned should be left as loosely coupled as possible, so that it can be removed if need be.